content/en/user/run-your-own.md

```diff
@@ -54,12 +56,14 @@ There exist a number of **dedicated Mastodon hosting providers** that take care
 
 Managed hosting solutions are great for those who do not have experience or desire to install and maintain software. However, being in charge of all components on your own hardware gives greater control over scaling, performance and customization.
 
-We provide a **DigitalOcean 1-Click Install Image** that you can put on a DigitalOcean droplet of your choosing which essentially gives you everything you would otherwise have after following our installation instructions through an interactive setup wizard.
+We provide a **DigitalOcean 1-Click Install Image** that you can put on a DigitalOcean droplet of your choosing, which essentially gives you everything you would otherwise have after following our installation instructions through an interactive setup wizard.
```

Comment:
```suggestion
We provide a **DigitalOcean 1-Click Install Image** that will deploy Mastodon on a DigitalOcean Droplet of your choosing, which essentially gives you everything you would otherwise have after following our installation instructions through an interactive setup wizard.
```